source/update.rst

```diff
@@ -37,6 +65,95 @@ The following steps will allow you to update both of them to current version
    $ kubectl create -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{{release}}}/deploy/operator.yaml
    $ kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
 
+
+
+.. _operator-update-smartupdates:
+
+Upgrading Percona Distribution for PostgreSQL
+---------------------------------------------
+
+Automatic upgrade
+*****************
+
+Starting from version 1.1.0, the Operator does fully automatic upgrades to
+the newer versions of Percona PostgreSQL Cluster within the method named *Smart
+Updates*.
+
+The Operator will carry on upgrades according to the following algorithm.
+It will query a special *Version Service* server at scheduled times to obtain
```

Comment:
We need to mention that VS has status  'tech preview ' now and it is disabled by default in 1.1.0.
@spron-in  Am I right?